src/executor/mod.rs

```diff
@@ -6,6 +6,8 @@
 mod stack;
 
 pub use self::stack::{
-	MemoryStackState, Precompile, PrecompileOutput, StackExecutor, StackExitKind, StackState,
-	StackSubstateMetadata,
+	MemoryStackAccount, MemoryStackState, MemoryStackSubstate, Precompile, PrecompileOutput,
+	StackExecutor, StackExitKind, StackState, StackSubstateMetadata,
 };
+
+pub use ethereum::Log;
```

Comment:
Why this though?

Reply:
Because of [this](https://github.com/gakonst/dapptools-rs/pull/59/files#diff-76f3a4d2f1d738d5e053b8f0b6a9014ff7b962ecbaa432ac949c8e2a512f4a43R223). Ideally I'd like to avoid having to import `ethereum` explicitly in my crate and have to manage its version & make sure it's in line with sputnik's.